### PR TITLE
Fixes reconfigure_spinnaker.sh to behave properly when not run as root.

### PR DESCRIPTION
**Pull request by ewiseblatt**
@duftler

### CODE REVIEW

**Comment by AMeng** [comment]
Thanks for this :smile:

File: pylib/spinnaker/configurator.py [hunk 1]
```diff
@@ -120,29 +120,35 @@ def __init__(self, installation_parameters=None, bindings=None):
...
-         # ambguous which we are validating. For saftey we'll force this
-         # to be the normal system installation. Warn that we are doing this.
-        user_config = os.path.join(os.environ['HOME'], '.spinnaker')
-        deck_dir = installation_parameters.DECK_INSTALL_DIR
-        if os.path.exists('/root/.spinnaker'):
-            user_config = '/root/.spinnaker'
+      if os.geteuid():        
+          # If we are not running as root and there is an installation on
+          # this machine as well as a user/.spinnaker directory then it is
+          # ambguous which we are validating. For saftey we'll force this
```
Comment (duftler) [inline pylib/spinnaker/configurator.py]: s/ambguous/ambiguous

**Comment by duftler** [comment]
Please don't merge this yet. I want to test it first.

**Comment by duftler** [comment]
Tested.
LGTM.

**Pull request merged**